xcdat/bounds.py

```diff
@@ -374,26 +427,12 @@ def _create_bounds(self, axis: CFAxisKey, coord_var: xr.DataArray) -> xr.DataArr
         diffs = np.insert(diffs, 0, diffs[0])
         diffs = np.append(diffs, diffs[-1])
 
-        # `cftime` objects only support arithmetic using `timedelta` objects, so
-        # the values of  `diffs` must be casted from `dtype="timedelta64[ns]"`
-        # to `timedelta` objects.
-        if axis == "T" and issubclass(type(coord_var.values[0]), cftime.datetime):
-            bounds = create_time_bounds(coord_var)
-            return bounds
-
         # width parameter: determines bounds location relative to midpoints
         width = 0.5
 
-        # FIXME: These lines produces the warning: `PerformanceWarning:
-        # Adding/subtracting object-dtype array to TimedeltaArray not
-        # vectorized` after converting diffs to `timedelta`. I (Tom) was not
-        # able to find an alternative, vectorized solution at the time of this
-        # implementation.
-        with warnings.catch_warnings():
-            warnings.simplefilter("ignore", category=pd.errors.PerformanceWarning)
```

Comment:
These lines are no longer needed since we aren't do `timedelta` arithmetic in `_create_bounds()`.